torch/onnx/utils.py

```diff
@@ -257,6 +257,9 @@ def _model_to_graph(model, args, f, verbose=False, training=False,
     param_names = input_and_param_names[len(input_and_param_names) - len(params):]
     params_dict = dict(zip(param_names, params))
 
+    if do_constant_folding:
+        params_dict = torch._C._jit_pass_onnx_constant_fold(graph, params_dict)
```

Comment:
yeah, we have to do this pass after we get the param_dict.

Reply:
Yes.